emacs/merlin.el

```diff
@@ -1949,7 +1949,7 @@ Runs a merlin process in the background and perform queries on it.
 
 Short cuts:
 \\{merlin-mode-map}"
-  nil
```

Comment:
Will this change remain compatible with older versions of Emacs ?
(sorry, I am not myself an Emacs user)

Reply:
Yes. `:init-value` was introduced in Emacs 21, along with the other keywords that merlin.el already uses.
Positional arguments to `define-minor-mode` have been deprecated for years but it's only now in Emacs 28 that we get a warning about them.